envs/example/defaults.yml

```diff
@@ -40,6 +40,11 @@ memcached:
   user: nobody
   max_connections: 1024
 
+swift:
+  enabled: False
+  haproxy_vip: "{{ swift_fqdn }}"
```

Comment:
If swift_fqdn is referenced in this file, it should also be defined in this file, otherwise there will be an undefined variable.